### DISCUSSION TITLE
Data update frequency of Osmand elevation widget

### DISCUSSION THREAD

**Discussion by claus2022** [osmandapp/OsmAnd#15233] [Q&A]
Testing the elevation widget during last week I often got confused by mismatch of the actual position shown on the profile and my real location. After observing the widget for a while I found as probable reason, that the elevation profile is not updated fast enough. Sometimes the actual data appear after serveral minutes, which is much to long for Mountainbiking.

Is there any possibility to make data updating faster?

**Comment by vshcherb** [comment]
Elevation widget displays altitude from track (map data) i.e. SRTM-data and it's not actual elevation from the GPS-device. As I understand data should be updated as fast as possible.

**Reply by claus2022** [reply]
Sorry, I explained it not clearly enough: My problem does not refer to the numerical altitude field shown in the "Right Panel"-widget. It refers to the graphical elevation profile shown at the bottom of the screen during navigation. This graph could be helpful for better orientation and mental preparation prior to steep climbs. It shows the course of elevation correctly, but my actual position within this graphical elevation profile is updated with delays of several minutes. These delays make the elevation profile worthless for both, for better orientation as well as for early recognition of approaching climbs.

**Comment by scaidermern** [comment]
Is it the same problem as #14700?
